backend/app/crud/tts_evaluations/cron.py

```
@@ -0,0 +1,546 @@
+            all_terminal = False
+            continue
+
+        # Batch reached terminal state - process it
+        if provider_status == BatchJobState.SUCCEEDED.value:
+            await process_completed_tts_batch(
+                session=session,
+                run=run,
+                batch_job=batch_job,
+                batch_provider=batch_provider,
+            )
+            any_succeeded = True
```

Ruff (0.15.1)

[error] Local variable `any_succeeded` is assigned to but never used

Remove assignment to unused variable `any_succeeded`

(F841)